BridgeSDK/Networking/SBBNetworkManager.m

```diff
@@ -218,6 +218,11 @@ - (instancetype) initWithBaseURL: (NSString*) baseURL bridgeStudy: (NSString*)br
         self.environment = SBBEnvironmentCustom;
```

Comment:
Yes, this is only a default setting. This initializer isn't exposed in the public API and callers reset it appropriately.